documentation/config_file_auth.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: auth.conf"
+canonical: "/puppetserver/latest/config_file_auth.html"
+---
+
+[`trapperkeeper-authorization`]: https://github.com/puppetlabs/trapperkeeper-authorization
```

Comment:
I do think the parts of `trapperkeeper-authorization` that are _relevant to Puppet Server_ should be on our docs site, and documented in the context of a Puppet Server install. If there's more settings required in `auth.conf`, for instance, we should document them in this `config_file_auth` document. 

If that means we need to add `sort-order` and `name` in addition to the changes in this PR, I can add those in a follow-up PR.

If there's any optional parts of `trapperkeeper-authorization` that _can_ work with Puppet Server but aren't _required_, I'm OK with linking to GitHub _or_ adding an "advanced options" section to the appropriate configuration file documents, as long as we emphasize that those options are not required.